front/app/components/admin/ActionForm/messages.ts

```diff
@@ -0,0 +1,79 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  none: {
+    id: 'app.containers.Admin.projects.project.permissions.components.PhasePermissionsNew.ActionsFormNew.none',
+    defaultMessage: 'None',
+  },
+  noneSubtitle: {
+    id: 'app.containers.Admin.projects.project.permissions.components.PhasePermissionsNew.ActionsFormNew.noneSubtitle',
+    defaultMessage: 'Anyone can participate without signing up or logging in.',
+  },
+  emailConfirmation: {
+    id: 'app.containers.Admin.projects.project.permissions.components.PhasePermissionsNew.ActionsFormNew.emailConfirmation',
+    defaultMessage: 'Email confirmation',
+  },
+  emailConfirmationSubtitle: {
+    id: 'app.containers.Admin.projects.project.permissions.components.PhasePermissionsNew.ActionsFormNew.emailConfirmationSubtitle',
+    defaultMessage:
+      'Participants need to confirm their email with a one-time code.',
+  },
+  accountCreation: {
+    id: 'app.containers.Admin.projects.project.permissions.components.PhasePermissionsNew.ActionsFormNew.accountCreation',
+    defaultMessage: 'Account creation',
+  },
+  accountCreationSubtitle: {
+    id: 'app.containers.Admin.projects.project.permissions.components.PhasePermissionsNew.ActionsFormNew.accountCreationSubtitle',
+    defaultMessage:
+      'Participants need to create a full account with their name, confirmed email and password.',
```

Comment:
Again a good catch, I added another message in case email confirmation is disabled. I've tried to keep it in mind, e.g. in the flow visualization and we hide the 'confirmed email' card button is it's disabled. But I added a ticket for tomorrow to test this explicitly: https://www.notion.so/govocal/Test-to-make-sure-things-make-sense-without-email-confirmation-enabled-8b1a60e798fc4b9faffe50fcfed0783b